src/build/tasks/style.ts

```diff
@@ -12,7 +12,7 @@ export interface InlineStylesheet {
   contents: string;
 }
 
-export async function createStyles(inlines: InlineStylesheet[], outputDir: string, sassDir: string) {
+export async function buildStyles(sassDir: string, outputDir: string, inlines: InlineStylesheet[] = []) {
```

Comment:
Updated this function to follow more intuitive signature "source, destination, options"